frappe/commands/utils.py

```diff
@@ -583,11 +583,30 @@ def auto_deploy(context, app, migrate=False, restart=False, remote='upstream'):
 	else:
 		print('No Updates')
 
+@click.command('disable-admin-2fa')
+@pass_context
+def two_fa(context):
+	'''Disable 2fa for admin.'''
+	for site in context.sites:
+		try:
+			frappe.init(site=site)
+			frappe.connect()
+			frappe.set_value(
+				"System Settings",
+				"enable_two_factor_authentication_for_administrator",
```

Comment:
this property must also exist in the System Settings doctype, and must have a default value too...